tests/verifyRemoteInstances_test.js

```diff
@@ -10,32 +10,57 @@ xScenario('Open Remote Instance Page and Add mysql instances @pmm-pre-update', a
-    remoteInstancesPage.createRemoteMySQL(mysql_service_name, version);
+    remoteInstancesPage.openAddRemotePage('mysql');
+    remoteInstancesPage.fillRemoteFields(mysql_service_name);
+    remoteInstancesPage.createNewRemoteInstance(mysql_service_name, version);
     pmmInventoryPage.verifyOldMySQLRemoteServiceIsDisplayed(mysql_service_name);
     await pmmInventoryPage.verifyAgentHasStatusRunning(mysql_service_name, version);
 });
 
-
 xScenario('Verify is the remote instances are in Running Status @pmm-post-update', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
     let mysql_service_name = "mysql_remote_test";
     let version = "new";
     I.amOnPage(pmmInventoryPage.url);
-    pmmInventoryPage.verifyMySQLRemoteServiceIsDisplayed(mysql_service_name);
+    pmmInventoryPage.verifyRemoteServiceIsDisplayed(mysql_service_name);
     await pmmInventoryPage.verifyAgentHasStatusRunning(mysql_service_name, version);
-
 });
 
 xScenario('Open Remote Instance Page and Add mysql instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
     let mysql_service_name = "mysql_remote_new";
     let version = "new";
     I.amOnPage(remoteInstancesPage.url);
     remoteInstancesPage.waitUntilNewRemoteInstancesPageLoaded();
-    remoteInstancesPage.openAddRemoteMySQLPage();
-    remoteInstancesPage.fillRemoteMySQLFields(mysql_service_name);
-    remoteInstancesPage.createRemoteMySQL(mysql_service_name, version);
-    pmmInventoryPage.verifyMySQLRemoteServiceIsDisplayed(mysql_service_name);
+    remoteInstancesPage.openAddRemotePage('mysql');
+    remoteInstancesPage.fillRemoteFields(mysql_service_name);
+    remoteInstancesPage.createNewRemoteInstance(mysql_service_name, version);
+    pmmInventoryPage.verifyRemoteServiceIsDisplayed(mysql_service_name);
     await pmmInventoryPage.verifyAgentHasStatusRunning(mysql_service_name, version);
+});
+
+xScenario('Open Remote Instance Page and Add MongoDB instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
+    let mongodb_service_name = "mongodb_remote_new";
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilNewRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('mongodb');
+    //need to be add method for create and verification after I got credentials
+});
 
+xScenario('Open Remote Instance Page and Add PostgreSQL instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
+    let postgresql_service_name = "postgresql_remote_new";
+    I.amOnPage(remoteInstancesPage.url);
+    remoteInstancesPage.waitUntilNewRemoteInstancesPageLoaded();
+    remoteInstancesPage.openAddRemotePage('postgresql');
+    //need to be add methods for create and verification after I got credentials
+});
+
+xScenario('Open Remote Instance Page and Add ProxySQL instances PMM Latest', async (I, adminPage, remoteInstancesPage, pmmInventoryPage) => {
```

Comment:
could you please rename these scenarios to one template like : Verify user is able to add ProxySQL instance ?